src/form-builder/components/FormBuilder.jsx

```diff
@@ -68,6 +74,66 @@ export default class FormBuilder extends Component {
     }, commonConstants.toastTimeout);
   }
 
+  getValidJsonFileNames(fileNames) {
+    const validJsonFiles = [];
+    fileNames.forEach(fileName => {
+      if (fileName.endsWith('.json')) {
+        validJsonFiles.push(fileName);
+      } else {
+        this.updateImportErrors(fileName, 'Invalid file format');
+      }
+    });
+    return validJsonFiles;
+  }
+
+  importJsonZip(jsonZip) {
+    const self = this;
+    const maxAllowedSize = 500 * 1024;
+    if (jsonZip.size > maxAllowedSize) {
+      self.hideLoader();
+      self.props.onValidationError('Error Importing.. Exceeded max file size 500KB');
+    } else {
+      const jsZip = new JSZip();
+      jsZip.loadAsync(jsonZip).then((zip) => {
+        self.validateExtractedZip(zip);
+      });
+    }
+  }
+
+  importForm(file) {
```

Comment:
I see we have 3 methods with similar name like 'importForm', 'importForms' and 'importFormJson' and its hard to figure it which method does which task. It would be good if we can come up with relevant method name